doc(pe): Document policy options (#601)
armory/docs: content/en/docs/plugin-guide/plugin-policy-engine/packages/spinnaker.http.authz/tasks/type.deployManifest.md

---
title: "Task Type: deployManifest"
linktitle: "deployManifest"

> description: "Policy controls whether or not a deployManifest that is triggered from outside a spinnaker pipeline (e.g. from the ‘Clusters’ tab of an application's 'edit' action) can run."

Reviewer: ```suggestion
description: "Policy that controls whether or not a deployManifest that is triggered from outside a  pipeline (such as from the **Clusters** tab of an application's **Edit** action) can run."
```

Reviewer: Wait actually does bold render in descriptions?
